processout-sdk/src/main/java/com/processout/processout_sdk/Transaction.java

```diff
@@ -0,0 +1,319 @@
+package com.processout.processout_sdk;
+
+import com.google.gson.annotations.SerializedName;
+
+class Transaction {
```

Comment:
Do we actually need this object? (is it used somewhere?)

Reply:
Yep it's used for the authorization result (the class is private so not available outside th SDK)